deployability/modules/allocation/allocation.py

```diff
@@ -3,6 +3,8 @@
 # This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
 
 import yaml
+import paramiko, logging, time
+import winrm
 
```

Comment:
Add `winrm` to requirements.txt file with a version

```
[2024-04-16 20:06:03] [DEBUG] [57246] [ThreadPoolExecutor-1_0] [workflow_engine]: Running task "allocate-manager-linux-ubuntu-22.04-amd64" with arguments: ['modules/allocation/main.py', '--action=delete', '--track-output=/tmp/dtt1-poc/manager-linux-ubuntu-22.04-amd64/track.yaml']
[2024-04-16 20:06:03] [ERROR] [57246] [ThreadPoolExecutor-1_0] [workflow_engine]: [allocate-manager-linux-ubuntu-22.04-amd64] Task failed with error: Error executing process task Traceback (most recent call last):
  File "/home/nonsatus/Documents/Wazuh/Repositories/wazuh-qa/wazuh-qa/deployability/modules/allocation/main.py", line 12, in <module>
    from modules.allocation import Allocator
  File "/home/nonsatus/Documents/Wazuh/Repositories/wazuh-qa/wazuh-qa/deployability/modules/__init__.py", line 6, in <module>
    from .allocation import Allocator
  File "/home/nonsatus/Documents/Wazuh/Repositories/wazuh-qa/wazuh-qa/deployability/modules/allocation/__init__.py", line 5, in <module>
    from .allocation import Allocator
  File "/home/nonsatus/Documents/Wazuh/Repositories/wazuh-qa/wazuh-qa/deployability/modules/allocation/allocation.py", line 7, in <module>
    import winrm
ModuleNotFoundError: No module named 'winrm'
```

When I tried to add it I get this error

```
ERROR: Could not find a version that satisfies the requirement winrm (from versions: none)
ERROR: No matching distribution found for winrm
```

Reply:
Done: 1ffd41c6b